Resources/ServerInfo/Guidebook/ServerRules/RoleplayRules/RuleR4-2AntagMetashield.xml

```diff
@@ -0,0 +1,124 @@
+  - discovering an unlocked uplink
+
+  ## Chameleon Items
+
+  Chameleon items are shielded.
+
+  Being suspicious of an item being fake or stolen is not shielded, but testing items or calling them chameleon is covered by this shield.
+
+  The revealing condition for this shield is any of the following:
+  - seeing someone else cause any chameleon item to change
+  - finding holographic nanomachine fibers
+  - experiencing a situation where absolutely no other explanation is possible
+  - discovering an unlocked uplink
+
+  ## Stealth Items
+
+  The fact that an item can be something other than what its visual appearance and examine description indicate is shielded.
+
+  This shield protects stealth items, including protecting them from being tested.
+
+  The revealing condition for this shield is any of the following:
+  - seeing the item behave differently than the expected behavior for the item
+  - seeing the item used for its hidden purpose
+  - experiencing a situation where absolutely no other explanation is possible
+  - discovering an unlocked uplink
+
+  ## Traitor
+
+  The existence of Syndicate agents who infiltrate and sabotage the station is not shielded.
+
+  ## Thief
+
+  The existence of kleptomaniacs who steal anything that is not bolted down is not shielded.
+
+  Knowledge of what items are possible thief targets is shielded, but knowledge that some items are valuable or would be dangerous in the wrong hands is not shielded. (ex: Captain's ID, Chief Engineer's magboots)
+
+  ## Ninja
+
+  The existence of Ninjas and the Spider Clan beyond being a pop culture mainstay is shielded.
+
+  The revealing condition for this shield is the following:
+  - the Ninja is witnessed cloaking, uncloaking or teleporting. Spotting an actively cloaked Ninja does not count.
+  - you witness an uncloaked Ninja who is aggressive or otherwise actively completing their objectives (emagging doors, tampering with computers)
+
+  ## Changeling
+
+  The existence of cannibal shapeshifters being real is shielded.
+
+  The revealing condition for this shield is any of the following:
+  - you are the Research Director or Chief Medical Officer
+  - you witness the Changeling eat or shapeshift
+  - the Changeling assaults you using its abilities
+  - you witness the Changeling use an ability that cannot otherwise be explained
+
+  The unique quality of Changeling blood to react violently in a centrifuge is shielded.
+
+  The revealing condition for this shield is the following:
+  - you are the Research Director or Chief Medical Officer
+
+  The ability to prevent the revival of a Changeling by cremating them is shielded.
+
+  The revealing condition for this shield is any the following:
+  - you are the Research Director or Chief Medical Officer
+  - you have witnessed a Changeling revive
+
+
+  ## Revolutionaries
+
+  The existence of union organizers and prospective mutineers is not shielded. The concept of a Syndicate-backed mutiny is also not shielded.
```

Comment:
im a little iffy on this because i like leaving room for rp on what being a revolutionary means. my flavor changes indicate that they are "removing NT brainwashing" w/ flashes but i like the ambiguity. if that makes sense. anyway its currently extraordinarily easy to counterplay revs by getting anyone who has a flash so im not really fond of protecting that. if that makes sense